src/widgets/splits/Split.hpp

```diff
@@ -77,6 +77,9 @@ class Split : public BaseWidget
 
     void setInputReply(const MessagePtr &reply);
 
+    // This is called on window focus lost
```

Comment:
Documentation should say what an entity does, not where it's used. It could say where it's supposed to be used.